.github/workflows/prod.yaml

```diff
@@ -63,7 +63,139 @@ jobs:
       - name: Build
         run: npm run test:functional:build:prod
       - name: Run TestCafe Tests
-        run: npx testcafe -q -c 5 'saucelabs:Chrome@latest:macOS 11.00','saucelabs:IE@latest:Windows 10','saucelabs:Firefox@latest:Windows 10','saucelabs:Safari@latest:macOS 11.00'
+        run: npx testcafe -q -c 10 "saucelabs:Chrome@latest:macOS 11.00",
+        env:
+          ALLOY_PROD_VERSION: ${{ steps.package-version.outputs.current-version }}
+      - uses: craftech-io/slack-action@v1
+        with:
+          slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
+          status: failure
+        if: failure()
+
+  safari_workflow:
```

Comment:
The GitHub actions documentation says you can have a multi-dimensional matrix and it will do every combination. https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

This could avoid a lot of the duplication here.

Reply:
I tried this first with no luck, the jobs are handled differently and ends up running the version with the test browser in a queue.